policyengine_canada/variables/gov/provinces/on/tax/income/credits/oeptc/is_senior_for_oeptc.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_canada.model_api import *
+
+
+class OEPTCSeniorStatus(Enum):
+    SENIOR = "Has at least one senior"
+    NON_SENIOR = "Has no seniors"
+
+
+class oeptc_senior_status(Variable):
+    value_type = Enum
+    entity = Household
+    label = "Is a senior for the OEPTC"
+    possible_values = OEPTCSeniorStatus
+    default_value = OEPTCSeniorStatus.NON_SENIOR
+    definition_period = YEAR
+
+    def formula(household, period, parameters):
+        oldest_age = household.members("age", period).max(axis=0)
```

Comment:
Unit test will reveal but I think this might have to be this instead:
```suggestion
        oldest_age = household.max(household.members("age", period))
```

Reply:
You were right, that solved it